Makefile

```diff
@@ -19,7 +19,7 @@ ifeq (${IMAGE_TAG},)
 IMAGE_TAG := ${SHORT_COMMIT}
 endif
 
-IMAGE_TAG_NO_NETGO := $(IMAGE_TAG)-without-netgo
+IMAGE_TAG_NO_NETGO := $(IMAGE_TAG)+without-netgo
```

Comment:
Nice!

I suggest to add the following test cases [here to validate](https://github.com/onflow/flow-go/blob/master/engine/execution/ingestion/stop/stop_control_test.go#L867)

```
	// a special build created with "+" would not change the version priority.
	require.True(t, semver.New("0.32.11+without-netgo").Equal(*semver.New("0.32.11")))
	require.True(t, semver.New("0.32.11-without-netgo").LessThan(*semver.New("0.32.11")))
```